openbb_platform/obbject_extensions/charting/pyproject.toml

```diff
@@ -9,7 +9,7 @@ packages = [{ include = "openbb_charting" }]
 
 [tool.poetry.dependencies]
 python = ">=3.9.21,<3.13"  # scipy forces python <4.0 explicitly
-openbb-core = "^1.4.3"
+openbb-core = "^1.4.6"
 pandas-ta-openbb = "^0.4.20"
 plotly = "^5.24.1"
```

Comment:
Another note:
plotly 6 has been out for a while now. We might want to look into it. I've not seen breaking changes on the graph object interface level. But there are some things under the hood that did impact how the workspace sees the data that is being used by plotly.js (specifically how AI sees that)
Poke me once you'll start looking into upgrading and I will share what we already have found.

Reply:
I had already looked at this and had to downgrade because the output of the serialized content is unexpected and was not layering correctly.

Reply:
What we found is that that behavior is only applied to numpy arrays and pandas series, so if you cast the array to a list, tuple or a set the base64 compression is not applied